src/app/layouts/components/header/header.component.scss

```diff
@@ -3,25 +3,25 @@
 
 .header {
     background: $white;
-    padding: 10px;
+    padding: 3px;
     width: 100%;
     display: flex;
-    font-size: 30px;
+    font-size: 11px;
     justify-content: space-between;
     align-items: center;
-    box-shadow: inset 0 0 15px 0 rgb(230, 228, 228);
+    box-shadow: inset 0 0 10px 0 rgb(230, 228, 228);
 
     .logo-img {
-        width: 100px;
-        height: 90px;
+        width: 60px;
```

Comment:
you are repeating these sizes in both headers, use variables for mobile, tablet and desktop